drape/overlay_handle.cpp

```diff
@@ -232,7 +232,7 @@ std::string SquareHandle::GetOverlayDebugInfo()
 /// @param[in] depth        Manual priority from styles (bigger is better)
 uint64_t CalculateOverlayPriority(int minZoomLevel, uint8_t rank, float depth)
 {
-  ASSERT(0 <= depth && depth <= 100000, (depth));
+  ASSERT(0 <= depth && depth <= 190000, (depth));
```

Comment:
A comment why 190K would be helpful.

Reply:
this function will need to be cleaned/refactored later to remove remnants of minVisibleScale anyway...

Reply:
it was actually a temporary rise for migration to work (some overlays had priorities as high as 182k);
now after priorities compression and re-spacing we can put back 100000 or even 10000 is more than enough